### PR TITLE
INTLY-3648 Add patch release process

### PR DESCRIPTION
**Pull request by david-martin**
## Additional Information
https://issues.jboss.org/browse/INTLY-3648

@maleck13 Anything sound off the rails in this?

### CODE REVIEW

**Comment by maleck13** [comment]
nope looks good

**Comment by david-martin** [comment]
Moved from draft to full PR now.
I've added info about the jira release dashboard and tagging the help repo (which could be moved to a script in that repo).

The diff is messy to compare, so probably easier to view the new version of the file (https://github.com/integr8ly/installation/blob/82eca843feecea76b18845da02f6a0411e63f4dc/docs/release.md).

cc @davidffrench for review please.

File: docs/release.md [hunk 1]
```diff
@@ -25,27 +60,39 @@ As this is a manual task, here are some guidelines for doing the review.
...
+3) Commit and push back any changes to the upstream
 
-    ``` ./scripts/release.sh -b v2.7 -r release-2.7.1-rc1```
-       
-            
+### Tagging the help repo
+
+Once a release has been signed off, the help repo will need to be tagged on the release branch:
+
+```
```
Comment (davidffrench) [inline docs/release.md]: Would be good to give a link to the help repo here.

Comment (david-martin) [reply docs/release.md]: Added. Can you re-review @davidffrench

**Pull request merged**